src/Form/ScheduleType.php

```diff
@@ -5,22 +5,28 @@
 namespace App\Form;
 
 use App\Dto\ScheduleRequest;
+use App\Service\Event\EventServiceInterface;
 use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 final class ScheduleType extends AbstractType
 {
+    private $eventService;
+
+    public function __construct(EventServiceInterface $eventService)
+    {
+        $this->eventService = $eventService;
+    }
+
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
-            ->add('day', DateType::class, [
+            ->add('day', ChoiceType::class, [
                 'required' => true,
-                'attr' => [],
-                'data' => new \DateTime('+ 1 day'),
```

Comment:
Are you sure you want to remove this ? Is it a rebase fail ?

Reply:
Yes because I Narrow down the new day possible by a list of choices, so I will send a dropdown list of all days possible in the daterange of the event, to lock the possibilities to create a day on the schedule that is outside the event date

Reply:
Ah ok I see.
Don't forget html5 datepicker with min and max attributes

Reply:
Be careful, there is no longer a list of choices now, it's a simple datepicker in html5

Reply:
Look at that : https://github.com/DarkmiraTour/community-event-manager/pull/158/commits/a4bf05dd4253f1f33f049db1ad71239393806ecb#diff-beb982adf4818730a38fbd7c3daa10faR18

Reply:
Yes but in this case, a datepicker that would make it possible to select only few says is maybe to much, that's why a select with some dates available (theses in the range of the event) is also a good solution